### PR TITLE
Count of printed documents

### PR DESCRIPTION
**Pull request by dwhenry**
Daily Postal Summary Document report

Display postal summary documents for the day so that
team leaders can check that all required documents
have been printed and are ready to be sent.

Team leaders are able to print any missing documents
directly from this page if they are missing.

### CODE REVIEW

File: features/reprint_summary_document.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Feature: Reprint Summary Document
+  As a user
+  I want to reprint a customers summary document
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: customer's

File: features/reprint_summary_document.feature [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Feature: Reprint Summary Document
+  As a user
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: Who's the user?

File: features/reprint_summary_document.feature [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  I want to reprint a customers summary document
+  So that I can resend it when the customer has not received the original
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: Are we resending or reprinting?

File: features/reprint_summary_document.feature [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  @unauthenticated
+  Scenario: Reprint a summary document
+    Given I log in to reprint a summary document
+    When I enter a customer reference number
+    And I select the matched record to reprint
+    Then I am able to edit the customers details
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: customer's

File: features/reprint_summary_document.feature [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Feature: Reprint Summary Document
+  As a user
+  I want to reprint a customers summary document
+  So that I can resend it when the customer has not received the original
+
+  @unauthenticated
+  Scenario: Reprint a summary document
+    Given I log in to reprint a summary document
+    When I enter a customer reference number
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: Is it a customer reference number? Or an appointment reference? How do they refer to it?

File: features/reprint_summary_document.feature [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    And I select the matched record to reprint
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: Something like `And I select the record matching the customer's details`?

File: features/reprint_summary_document.feature [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    Then I am able to edit the customers details
+    And I can reprint the customers summary document
+
+  @unauthenticated
+  Scenario: Postal summary document requests report
+    Given I log in to see postal summary documents report
```
Comment (benbarnett) [inline features/reprint_summary_document.feature]: `To see a report of summary documents sent by post` ?

**Comment by benbarnett** [comment]
Does https://github.com/guidance-guarantee-programme/output/pull/227/commits/ae49192635f1d4fc31426be5ae815c5c6e891c3d belong in this PR?

**Comment by dwhenry** [comment]
@benbarnett Features updated with suggestions. Are you happy to have this merged?

**Comment by benbarnett** [comment]
Yep! That's much clearer, nice one 💯

**Pull request merged**